doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,161 +3,177 @@
 Chapel Quickstart Instructions
 ==============================
 
-These instructions are designed to help you get started with
-a source distribution of the latest Chapel release.
+These instructions are designed to help users get started with a
+single-locale (shared-memory) implementation of Chapel from the source
+distribution of the release as quickly as possible.  If you are
+interested in taking the time to understand Chapel's configuration
+options, build process, and installation more thoroughly, please refer
+to :ref:`readme-chplenv` and :ref:`readme-building` instead.
 
-In the following instructions, note that building and using Chapel as
-described in steps 2-5 disables optional and advanced features in the
-interest of getting you a clean build as quickly as possible. Later
-sections explain how to re-build in the preferred configuration and how to
-enable more features, such as distributed memory execution.
+These instructions first have you build a minimal, stripped-down
+version of Chapel to reduce build times and the potential for
+third-party portability issues.  Once you are interested in a
+full-featured version of Chapel, refer to
+:ref:`using-a-more-full-featured-chapel` below.
 
 
-0) See :ref:`prereqs.rst <readme-prereqs>` for more information about system
-   tools and packages you may need to have installed to build and run Chapel.
+0) See :ref:`readme-prereqs` for information about system tools and
+   packages you should have available to build and run Chapel.
 
-1) If you don't already have Chapel 1.25, see
-   https://chapel-lang.org/download.html .
 
-2) If you are using a source release, build Chapel in a *quickstart*
-   configuration. The *quickstart* script used below is designed to help you
-   get started with Chapel. After that, you may want to switch to a more
-   full-featured configuration. See using-a-more-full-featured-chapel_ below.
+1) If you don't already have the Chapel 1.25 source release, see
+   https://chapel-lang.org/download.html.
 
-   a. Expand the source release if you haven't already:
+
+2) Build Chapel in its 'Quickstart' configuration:
+
+   a. Unpack the source release if you haven't already:
 
       .. code-block:: bash
 
          tar xzf chapel-1.25.0.tar.gz
 
-   b. Make sure that you are in the directory you just created by expanding the
-      source release, for example:
+   b. Make sure that you are in the directory that was created when
+      unpacking the source release, for example:
 
       .. code-block:: bash
 
          cd chapel-1.25.0
 
    c. Set up your environment for Chapel's Quickstart mode.
-      If you are using a shell other than bash,
-      see quickstart-with-other-shells_ below.
+      If you are using a shell other than ``bash``,
```

Comment:
FWIW, zsh also can use the same script (as also noted under `quickstart-with-other-shells`)

Reply:
Good point.  So if I'm interpreting you correctly, you're suggesting this should say "other than `bash` or `zsh`"?

Reply:
Yep.